bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
the bin/setup file is the "rails  way" to setup projects.... i'd create a ticket to merge scritpts/first_install into bin/setup after this PR got merged